test/jdk/java/awt/Desktop/JMenuItemSetAcceleratorTest.java

```diff
@@ -0,0 +1,89 @@
+
+import java.awt.Desktop;
+import java.awt.Desktop.Action;
+import java.awt.EventQueue;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.awt.event.ActionEvent;
+import javax.swing.SwingUtilities;
+import javax.swing.KeyStroke;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuBar;
+import javax.swing.JMenuItem;
+
+/*
+ * @test
+ * @key headful
+ * @bug 8296275
+ * @summary To verify the setAccelerator method of JMenuItem.
+ * @requires (os.family=="mac")
+ * @run main JMenuItemSetAcceleratorTest
+ */
+
+public class JMenuItemSetAcceleratorTest {
+    private static JFrame frame = new JFrame();
+    private volatile static CountDownLatch actionPerformLatch;
+    private static Robot robot;
+
+    private static void createAndShow() {
+        JMenuBar bar = new JMenuBar();
+        JMenu menu = new JMenu("File");
+        JMenuItem menuItem = new JMenuItem("Menu Item");
+        menuItem.setAccelerator(
+            KeyStroke.getKeyStroke(KeyEvent.VK_M, ActionEvent.META_MASK));
+        menuItem.addActionListener(e -> {
+            System.out.println("menu item action.");
+            actionPerformLatch.countDown();
+        });
+        menu.add(menuItem);
+        bar.add(menu);
+
+        frame.setJMenuBar(bar);
+        frame.setBounds(200, 200, 200, 200);
```

Comment:
Position the frame to the middle of the screen frame.setLocationRelativeTo(null);